src/main/java/teammates/ui/controller/InstructorCourseEditSaveAction.java

```diff
@@ -14,6 +15,11 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID);
         Assumption.assertPostParamNotNull(Const.ParamsNames.COURSE_ID, courseId);
 
+        // Restore previous attributes of the course
```

Comment:
"Retrieve" would be a better word than "Restore"

Reply:
Sure. I'll change that